README.md

```diff
@@ -82,3 +85,25 @@ Version `2.0.0` removes the `environment` variable and adds `web_acl_metric_name
 Version `2.0.0` replaces the `ip_set` variable with a `ip_sets` list variable, which accepts a list of `aws_wafregional_ipset` ids.  This variable allows the Web ACL to pull from multiple lists of blocked ip addresses, such that you can combine a global blocked list, and application-specific lists.  For example: `ip_sets = [resource.aws_wafregional_ipset.global.id, resource.aws_wafregional_ipset.helloworld.id]`.
 
 During the initial upgrade to `2.0.0`, and if you add additional dynamic rules, you'll need to delete your web ACLs, as terraform cannot properly handle peer dependencies between Rules and Web ACLs.  For convenience, you can use the `delete-web-acl` script in the scripts folder.  For example: `scripts/delete-web-acl WEB_ACL_ID`.  Once the Web ACL is deleted use terraform apply to recreate the Web ACL and associate with your resources as you had before.  Deleting a Web ACL does not delete any associated resources, such as Application Load Balancers; however, it will leave the resources temporarily unprotected.
+
+### 2.0.0 to 2.1.0
```

Comment:
It looks like we order new upgrade paths above older ones See
https://github.com/trussworks/terraform-aws-logs#upgrade-paths